packages/contracts/src/core/permission/PermissionManager.sol

```diff
@@ -89,10 +88,9 @@ contract PermissionManager is Initializable {
     );
 
     /// @notice A modifier to be used to check permissions on a target contract.
```

Comment:
does `target` make sense anymore ? it will always be `address(this)`.

Reply:
Good catch. Fixed in 91b8fccfb80b5224c781691ba6bd8e638efd1534